xml-parser/src/main/java/org/uniprot/core/xml/uniref/AbstractMemberConverter.java

```diff
@@ -174,5 +185,12 @@ protected void updateMemberFromXml(
                     break;
             }
         }
+
+        UniRefMemberIdType memberType =
+                UniRefMemberIdType.typeOf(xmlObj.getDbReference().getType());
+        if (memberType.equals(UniRefMemberIdType.UNIPROTKB)) {
+            memberType = getUniProtKBIdType(memberId, accession);
+        }
```

Comment:
We should comment the logic that how we are finding if a member if trembl or swissprot. Probably a comment in method getUniProtKBIdType.

Reply:
added a java doc for method getUniProtKBIdType